webapp/src/containers/MapContainer.tsx

```diff
@@ -146,6 +148,7 @@ const MapContainer: FunctionComponent<MapContainerProps> = ({ match, history })
       )
       .then(response => {
         dataPoints = dataPoints.concat(response.data.datapoints);
+        response.data.tickFormat[2] = response.data.tickFormat[2] * 1000 * 60
```

Comment:
Add const MINUTE = 1000 * 60 

```suggestion
        response.data.tickFormat[2] = response.data.tickFormat[2] * MINUTE
```